src/components/DataDecorator/Decorator/Decorator.js

```diff
@@ -37,25 +37,29 @@ class Decorator extends Component {
     this.props.onContextMenu(event, this.props.type, this.props.value);
   };
 
-  renderDecorator = (noIcon, path, inline, score, scoreThresholds) => (
-    <Fragment>
-      {!noIcon && (
-        <span className={`${namespace}__icon`}>
-          <Icon
-            fillRule="evenodd"
-            path={path}
-            size={inline ? 12 : 16}
-            viewBox="0 0 16 16"
-            title={this.props.scoreDescription(score, scoreThresholds)}
-          />
+  renderDecorator = (noIcon, path, inline, score, scoreThresholds) => {
+    const { scoreDescription, title, type, value } = this.props;
```

Comment:
There's a lot going of destructuring happening here and around line 66 (with some of those destructured props from line 66 being passed to this `renderDecorator`). If this PR is going to include some cleanup, can it be done to include the destructure around line 66?

Reply:
Good catch!